frontend/src/store/actions.js

```diff
@@ -55,7 +55,7 @@ export const actions = {
   [LOGOUT]({ commit }) {
     commit(mutations.SET_USER, null);
     deleteStorageData();
-    commit(mutations.SET_LOGOUT_PROGRESS, true);
+    // commit(mutations.SET_LOGOUT_PROGRESS, true);
```

Comment:
Is that some leftover from testing?

Reply:
I removed logout progress from the store since it's no longer needed